### DISCUSSION TITLE
My Git clone is not working

### DISCUSSION THREAD

**Discussion by Mr-Prince16** [community/community#53515] [Repositories]
### Select Topic Area

Bug

### Body

srinj@LAPTOP-BN07C645 MINGW64 /c/Angular (master)
$ git clone https://github.com/Mr-Prince16/HealthScorePredictor.git
Cloning into 'HealthScorePredictor'...

srinj@LAPTOP-BN07C645 MINGW64 /c/Angular (master)
 This is skipping and going into other command please help me out

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Mr-Prince16! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, you are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction. Since you're seeing an error when cloning your repository, I would also suggest checking out our troubleshooting doc here: [Troubleshooting cloning errors](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository#troubleshooting-cloning-errors).

I've gone ahead and moved this to the correct category for you. Good luck!
